File: SocketChat/SocketChat.DataAccess/ChitChatContext.vb
Imports Microsoft.EntityFrameworkCore
Imports SocketChat.Model

Public Class ChitChatContext : Inherits DbContext

    Public Overridable Property Users As DbSet(Of User)
    Public Overridable Property Hubs As DbSet(Of Hub)
    Public Overridable Property Messages As DbSet(Of Message)
    Public Overridable Property Roles As DbSet(Of Role)
    Public Overridable Property UserRoles As DbSet(Of UserRole)

    Protected Overrides Sub OnConfiguring(optionsBuilder As DbContextOptionsBuilder)
        MyBase.OnConfiguring(optionsBuilder)

        optionsBuilder.UseSqlServer("Server=10.19.10.42,1433;Initial Catalog=ChitChat;Persist Security Info=False;User ID=sa;Password=hehe123;Encrypt=False;TrustServerCertificate=True;Connection Timeout=30;")


    End Sub

    Protected Overrides Sub OnModelCreating(modelBuilder As ModelBuilder)
        MyBase.OnModelCreating(modelBuilder)

    End Sub


End Class

File: SocketChat/Context/ChitChatContext.cs
using Microsoft.EntityFrameworkCore;

public class ChitChatContext : DbContext
{
    public virtual DbSet<User> Users { get; set; }
    public virtual DbSet<Hub> Hubs { get; set; }

    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
    {
        base.OnConfiguring(optionsBuilder);

        optionsBuilder.UseSqlServer("Data Source=DESKTOP-R04PVQ3\\SQLEXPRESS01; Initial Catalog=ChitChat; Integrated Security=true; TrustServerCertificate=True");

        int[] ses = new int[] { 1, 2, 3 };  

        

    }
}
